django_project/certification/models/course_type.py

```diff
@@ -41,7 +41,7 @@ class CourseType(models.Model):
 
     description = models.TextField(
         help_text=_('Course type description.'),
```

Comment:
Course type description - 1000 character limit

Reply:
done @timlinux 
here is what look like in forms
![screenshot from 2018-02-07 17-46-55](https://user-images.githubusercontent.com/26101337/35912447-315b8eec-0c2f-11e8-8efe-527e1abdf1e8.png)